src/css/footer.scss

```diff
@@ -1,80 +1,142 @@
 footer {
-    flex-shrink: 0;
-    display: flex;
-    flex-direction: column;
-    align-items: stretch;
-    padding: 45px 0 0;
-    margin: 0 $gutter-width/2;
-    overflow: hidden;
-    border-top: 1px solid $black;
+    background-color: $black;
+    color: $white;
+    padding: var(--baseline-grid-height-2x) 0;
+
+    .col-sm-12 {
+        display: flex;
+    }
+
+    .footer-logo {
+        width: 10%;
+
+        svg {
+            transform-origin: top left;
+            transform: rotate(-90deg) translate(-100%, 0);
+
+            path {
+                fill: $white;
+            }
+        }
+    }
 
     .footer-menu {
+        padding-left: 10px;
+        width: 90%;
+        display: flex;
+        flex-wrap: wrap;
+
         ul {
-            display: flex;
-            flex-direction: column;
             padding: 0;
-            margin: 40px 0;
+            width: 37.5%;
+            margin: 0;
 
             li {
                 list-style: none;
-                margin: 1rem 0;
+                margin: var(--baseline-grid-height) 0;
 
                 a {
+                    color: $white;
+                    font-family: 'apercu-mono-regular-pro', sans-serif;
                     text-transform: uppercase;
-                    letter-spacing: 1.5px;
                     font-weight: 500;
+                    letter-spacing: 1.5px;
+                    position: relative;
+                    white-space: nowrap;
+
+                    &.icon-link {
+                        padding-left: 50px;
+
+                        img {
+                            position: absolute;
+                            left: 0;
+                            top: 50%;
+                            transform: translateY(-50%);
+                        }
+                    }
+                }
+
+                &:first-of-type {
+                    margin-top: calc(0.5 * var(--baseline-grid-height));
                 }
-            }
 
-            .copyright {
-                margin: 60px 0 1rem
+                &:nth-of-type(2) {
+                    a {
+                        img {
+                            left: -7px;
+                            top: 6px;
```

Comment:
This seems like an odd place to override `top`---is it targeting a particular image? Flagging mostly because that could lead to issues if anyone edits this part in the CMS.

Reply:
It's targeting the token icon for the "Mint tBTC" link. 😬 Sorry! I knew it wasn't the best move but my figma fu was not up to the task of re-slicing the icons so they were all the same size/aspect ratio.

Reply:
Yah no worries. Something to consider how to handle, I think, nothing more.